src/mscorlib/src/System/Diagnostics/Debug.Unix.cs

```diff
@@ -22,16 +18,11 @@ private static void ShowAssertDialog(string stackTrace, string message, string d
             }
             else
             {
-                // TODO: #3708 Determine if/how to put up a dialog instead.
-                var exc = new DebugAssertException(message, detailMessage, stackTrace);
-                if (!s_shouldWriteToStdErr) 
-                {
-                    // We always want to print out Debug.Assert failures to stderr, even if
-                    // !s_shouldWriteToStdErr, so if it wouldn't have been printed in
-                    // WriteCore (only when s_shouldWriteToStdErr), print it here.
-                    WriteToStderr(exc.Message);
```

Comment:
As you see if COMPlus_DebugWriteToStdErr=1 it would go to stderr. I don't know why we were always writing it here, and I don't know why I removed it. We could put it back. Does it go to eg CI logs, and the FF message doesn't?